### ISSUE TITLE
/download doesn’t offer Multipass

### ISSUE THREAD

**Issue by matthewpaulthomas** [canonical-web-and-design/ubuntu.com#5303]
[ubuntu.com/download](https://www.ubuntu.com/download) currently does not suggest Multipass as an alternative to downloading Ubuntu Server/Cloud manually. (#5277 is the counterpart issue for /download/server.)

For now, Multipass should be promoted as a slim card below the Server and Cloud cards.

![-download sketch](https://user-images.githubusercontent.com/19801137/59361385-abfa6480-8d29-11e9-97e5-84dd3f7617c6.png)

As posted in [the copydoc](https://docs.google.com/document/d/1egsCcmFiz8czeHsvwoB9Jz815Ij5Urtf7Ngrru6Zd4o/edit#heading=h.5z1omrrlnk2g), the card’s text should be: “Using Ubuntu in virtual machines? Try [Multipass](https://multipass.run/) for a mini-cloud of instant Ubuntu VMs on Linux, Windows, or macOS.”

This issue covers visual design and implementation on ubuntu.com.

For the visual design, there are two factors:

- Multipass should be visible as a new and interesting thing on the page. This might involve something as simple as making Multipass the only option with an icon:

  ![-download-with-icon](https://user-images.githubusercontent.com/19801137/59358573-b8c88980-8d24-11e9-8f3f-86db0a95a308.png)

  Or some kind of “New” ribbon:

  ![-download-with-ribbon](https://user-images.githubusercontent.com/19801137/59359578-799b3800-8d26-11e9-9759-e818757ce215.png)

- Multipass should visibly be associated with the Server and Cloud options, and not the Desktop, Flavours, or IoT options. This probably involves spacing between the groups. It might also involve a Suru pattern that spans the group(s):

  ![-download-with-suru](https://user-images.githubusercontent.com/19801137/59359461-49539980-8d26-11e9-89ed-93541c43cc20.png)

  Or a background tint for the cards in each group:

  ![-download-with-tints](https://user-images.githubusercontent.com/19801137/59359703-bcf5a680-8d26-11e9-9293-7c44de10c811.png)

(Assigned at request of @pmahnke. The design was tracked in [multipass-design#42](https://github.com/CanonicalLtd/multipass-design/issues/42).)

**Comment by anasereijo** [comment]
Thanks, @matthewpaulthomas it's looking great and very helpful for me to make the final visual design. :)

@SirSamTumless do you know what's the deadline for this? I don't think it's planned for this iteration and I feel I already have too much. Would be great to have some time to design this maybe next iteration?

**Comment by anasereijo** [comment]
Thanks, @matthewpaulthomas it's looking great and very helpful for me to make the final visual design. :)

@SirSamTumless do you know what's the deadline for this? I don't think it's planned for this iteration and I feel I already have too much. Would be great to have some time to design this maybe next iteration?

**Comment by SirSamTumless** [comment]
@anasereijo, I don't think that we do. I will add it to the proposed workload for the next iteration.

**Comment by anthonydillon** [comment]
Are we still waiting for v1 of multipass to be released?

**Comment by SirSamTumless** [comment]
@anasereijo This estimate seems a bit high to me - what do you think?
